doc/source/tutorial/interpolate/2D_fill_from_border.rst

```diff
@@ -0,0 +1,408 @@
+.. _tutorial-intepolate_2dfillfromborder:
+
+.. currentmodule:: scipy.interpolate
+
+==============================
+Filling an image from a border
+==============================
+
+Suppose we have values for a function only on the border of a region,
+either to estimate the background of an image from the parts not
+covered by the sample, or as part of a boundary-value problem.
+
+In particular, consider a square with sides of length :math:`L`, with
+function values on the boundary given by
+
+.. math::
+
+   \sin(6\pi x/L), y = 0 \text{or} y = L
+   \sin(6\pi y/L), x = 0 \text{or} x = L
+
+That is, a sine wave with three full periods on each side, identical
+on opposite sides of the square.
+
+------------------
+Analytic solutions
+------------------
+
+With this simple boundary condition, we can find a few analytic
+solutions as a baseline.
```

Comment:
I'm not entirely sure what is the problem these are solutions for. You mentioned estimating the background above, maybe expand it a little? This whole story started from a problem, maybe discuss it a bit? Even if you start from a real-world case and then quickly turn into a simplified artificial example.